File: HealthSysHub.Web.Pharmacy.UI/src/app/models/activateorinactivateuser.ts
export interface ActivateOrInActivateUser  {
  id?: string; // Use string for GUID representation in TypeScript
  hospitalId?: string; // Use string for GUID representation in TypeScript
  staffId?: string; // Use string for GUID representation in TypeScript
  labId?: string; // Use string for GUID representation in TypeScript
  pharmacyId?: string; // Use string for GUID representation in TypeScript
  isActive: boolean;
  modifiedBy?: string; // Use string for GUID representation in TypeScript
  modifiedOn?: Date; // Use Date for DateTimeOffset representation in TypeScript
}

File: HealthSysHub.Web.Utility/Models/ActivateOrInActivateUser.cs
namespace HealthSysHub.Web.Utility.Models
{
    public class ActivateOrInActivateUser
    {
        public Guid? Id { get; set; }
        public Guid? HospitalId { get; set; }
        public Guid? StaffId { get; set; }
        public Guid? LabId { get; set; }
        public Guid? PharmacyId { get; set; }
        public bool IsActive { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
    }
}
